:globe_with_meridians: Add Spanish translation (#65)
zenika-open-source/open-source-promotion-cheat-sheet: README-es.md

<p align="center">
    <img alt="oss image" src="./imgs/zoss-logo.svg" height="200px" width="200px">
</p>

<p align="center">
  <a href="https://twitter.com/intent/tweet?text=How%20to%20promote%20your%20open-source%20projects%20@ZenikaOSS&url=https://github.com/zenika-open-source/open-source-promotion-cheat-sheet&hashtags=OpenSource,CheatSheet">

>     <img alt="tweet" src="https://img.shields.io/twitter/url/https/twitter?label=Share%20on%20twitter&style=social" target="_blank" />

Reviewer: Could you trasnlate "Share on twitter" ?

Author: Of course!

  </a>
</p>

<h1 align="center">Hoja de referencia de promoción al código abierto</h1>

> Esta hoja de referencia resume los pasos importantes que debes seguir para promover tu proyecto de código abierto en óptimas condiciones. Puedes hacer clic en los elementos para expandir y obtener más información de un tema.

Lista de lenguajes disponibles:

> - [Inglés](./README.md)

Reviewer: No need to translate these countries in Spanish (except Spanish) 😉

Author: Okok

- [Frances](./README-fr.md)
- [Alemán](./README-de.md)
- [Español](./README-es.md)

¿Un lenguaje falta? ¿Deseas mejorar esta hoja? ¡[Las contribuciones son bienvenidas](./CONTRIBUTING.md)!

## 1. 🎢 Pasos de preparación

<br />

<details>
<summary>👌 Asegúrate que tu proyecto sea lo suficientemente maduro</summary>
<p>

> Tu proyecto debe ser lo suficientemente estable con las características mínimas viables con el fin de atraer a los usuarios.

</p>
</details>

<details>
<summary>😎 Escoge un nombre atractivo para tu proyecto</summary>
<p>

> Escoge un nombre que los usuarios puedan recordar fácilmente.

</p>
</details>

<details>
<summary>💅 Haz que tu README sea elegante</summary>
<p>

> El README es lo primero que verán tus visitantes. Hazlo simple, elegante y fácil de leer. [Aquí hay una lista de ejemplos README](https://github.com/matiassingers/awesome-readme).

</p>
</details>

<details>
<summary>💪 Destaca los puntos fuertes de tu proyecto</summary>
<p>

> Identifica las fortalezas de tu proyecto y asegúrate de que los visitantes las vean primero.

</p>
</details>

<details>
<summary>✨ Proporciona una demostración de tu proyecto</summary>
<p>

> > los visitantes desean comprender rápidamente el propósito de tu proyecto, cómo funciona y cómo usarlo. Proporciona una demostración, es la mejor manera de complacer a los usuarios. Podría ser:

Reviewer 2: ```suggestion
> Los visitantes desean comprender rápidamente el propósito de tu proyecto, cómo funciona y cómo usarlo. Proporciona una demostración es la mejor manera de complacer a los usuarios. Podría ser:
```

Author: Thanks for the observation

>
>  - Un GIF animado que demuestra cómo funciona tu proyecto
>  - Un enlace a una demostración

</p>
</details>

<details>
<summary>👌 Instalación/Manejo debe ser súper fácil</summary>
<p>

> Probablemente perderás visitantes si tu proyecto no es fácil de usar.

</p>
</details>

<details>

> <summary>📘 Crea una documentación limpia y bien estructurada.</summary>

Reviewer 2: ```suggestion
<summary>📘 Crea una documentación limpia y bien estructurada</summary>
```

<p>

> Crear una buena documentación es probablemente el paso más importante. Si tienes una pequeña documentación, puedes incluirla en tu archivo README. De lo contrario, probablemente deberías alojarlo en un sitio web por separado. Algunos proyectos de código abierto como [vuepress](https://v1.vuepress.vuejs.org) pueden ayudarte a crear documentación limpia de una manera simple.

</p>
</details>

<br />

## 2. 📢 Correr la voz sobre tu trabajo

<br />

<details>
<summary>⭐ Haz que tu proyecto sea confiable antes de publicarlo en las redes sociales</summary>
<p>

> La mayoría de los visitantes comprobarán cuántas estrellas tiene tu proyecto antes de considerar usarlo. Una cantidad mínima de estrellas hace que tu proyecto sea más confiable que un proyecto con cero estrellas. Es por eso que debes pedir a las personas que conoces que apoyen tu proyecto antes de hacer un anuncio público en las redes sociales.

</p>
</details>

<details>

> <summary>↗️ Comparte tu proyecto en redes sociales y plataformas especializadas.</summary>

Reviewer 2: ```suggestion
<summary>↗️ Comparte tu proyecto en redes sociales y plataformas especializadas</summary>
```

</p>
</details>

<details>
<summary>🕐 Elige el mejor momento para publicar en las redes sociales</summary>
<p>

> No publiques durante el período de vacaciones o fines de semana. Por lo general, el mejor momento para publicar en las redes sociales es a mitad de semana.

</p>
</details>

<details>
<summary>🗑 No hagas spam en plataformas con tu mención</summary>
<p>

> No publiques dos veces en la misma plataforma. Es considerado como spam y puede causar mala publicidad para tu proyecto.

</p>
</details>

<br />

## 3. 🤝 Conserva a tus usuarios

<br />

<details>
<summary>🆕 Intenta lanzar regularmente nuevas versiones de tu proyecto</summary>
<p>

> Realiza mantenimiento y mejora tu proyecto con nuevos lanzamientos y genera un registro de cambios.

</p>
</details>

<details>
<summary>❗ Manten tu proyecto, revisa los problemas pendientes</summary>
<p>

> No dejes problemas pendientes sin respuesta. Sé amable con las personas que se tomaron el tiempo para dejar una sugerencia. 😉

</p>
</details>

<details>
<summary>🙏 Invita a otros usuarios a contribuir</summary>
<p>

> > Un proyecto saludable es un proyecto con una comunidad y contribuidores activos. Informa a tus usuarios que necesita ayuda etiquetando algunos problemas con las etiquetas de  `contribution welcome` o `good first issue`. [Ver más etiquetas de github](https://help.github.com/en/articles/about-labels).

Reviewer 2: ```suggestion
> Un proyecto saludable es un proyecto con una comunidad y contribuidores activos. Informa a tus usuarios que necesita ayuda etiquetando algunos problemas con las etiquetas de `contribution welcome` o `good first issue`. [Ver más etiquetas de github](https://help.github.com/es/github/managing-your-work-on-github/about-labels).
```

</p>
</details>

<details>
<summary>🏆 Recompensas a los contribuidores</summary>
<p>

> > ¡Sé amable con las personas que te ayudaron! Algunos proyectos de código abierto como [gatsby](https://github.com/gatsbyjs/gatsby) premian a los contribuidores. Si no puedes pagar eso, haz una publicación (en twitter u otras plataformas) sobre la contribución y menciona al autor ([este es un ejemplo de agradecimiento público](https://twitter.com/FranckAbgrall/status/1139470547492978688 )). Abre una sección de `Contribuidores` en tu archivo README para agradecerles públicamente o mostrarlos en la documentación de tu proyecto o sitio web. Aquí hay unos ejemplos:

Reviewer 2: ```suggestion
> ¡Sé amable con las personas que te ayudaron! Algunos proyectos de código abierto como [gatsby](https://github.com/gatsbyjs/gatsby) premian a los contribuidores. Si no puedes pagar eso, haz una publicación (en twitter u otras plataformas) sobre la contribución y menciona al autor ([este es un ejemplo de agradecimiento público](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Abre una sección de `Contribuidores` en tu archivo README para agradecerles públicamente o mostrarlos en la documentación de tu proyecto o sitio web. Aquí hay unos ejemplos:
```

>
> - [vuepress (contribuidores en la sección README)](https://github.com/vuejs/vuepress#code-contributors)
> - [Rythm.js (contribuidores aleatorios destacados en página web)](https://okazari.github.io/Rythm.js/)

</p>
</details>

<details>
<summary>💬 Abre una plataforma de chat comunitario</summary>
<p>

> Los problemas pendientes en Github no siempre son la mejor manera de comunicarte con tus usuarios. Si es necesario, puedes usar plataformas de chat para discutir con ellos.
>
> - [Discord](https://discordapp.com)
> - [Slack](https://slack.com)
> - [Gitter](https://gitter.im/)

</p>
</details>

<details>
<summary>🔙 Pide retroalimentación</summary>
<p>

> Los comentarios de los usuarios son la mejor manera de mejorar tu proyecto. Probablemente tienen características e ideas que podrían mejorar tu proyecto.

</p>
</details>

<details>
<summary>❤️ Abre una galería de casos de uso para mostrar a los usuarios que utilizaron tu proyecto</summary>
<p>

> Los visitantes confiarán en tu proyecto si ven casos de uso concretos e historias de éxito, por ejemplo, [la gallería vuepress](https://vuepress.gallery/).

</p>
</details>

<br />

## 🙏 Muestra tu apoyo

> ⭐️ a este repositorio ¡si esta hoja de referencia te ayudo!

Reviewer 2: **te ayuda** or **te ayudó**?

Author: Small details matter

## 🙏 Muestra tu apoyo

> ⭐️ a este repositorio ¡si esta hoja de referencia te ayudó!

Author: Small details matter.
